src/main/java/pl/ateam/disasteralerts/user/UserFacade.java

```diff
@@ -24,4 +25,12 @@ public Set<UserDTO> getInterestedUsers(String location) {
     public Set<UserDTO> getAllUsersByLocation(String location) {
         return userService.findAllByLocation(location);
     }
+
+    public UserUpdateDTO getUserForUpdate(UUID userId) {
+        return userService.findUserUpdateDto(userId);
+    }
+
+    public void updateUser(UserUpdateDTO userUpdateDto, UUID userId) {
+        userService.updateUserEntity(userUpdateDto, userId);
+    }
 }
```

Comment:
Czy parametr userId mogłby być częścią UserUpdateDTO czy to wynika z tworzenia tego obiektu na froncie?

Reply:
UserId pochodzi z principal. Zabezpiecza to przed tym, że ktoś może na froncie podmienić id i wysłać post, bo wtedy zmieniłby dane dla losowego użytkownika, a tak zawsze zmieniłby tylko swoje.